README.md

```diff
@@ -101,23 +101,23 @@ It comes with a prepopulated SQLite database.
 	- Note: Construals depend on supersenses, so if there are any supersenses missing from the database, you may get a 'Missing Supersense' error.
 - Import new usages:
-	- Run `python manage.py shell` and then type `exec(open(r'..\scripts\generate_usage_file.py').read())`. This will create a json file for all usages in the corpus. 
-	- You can then import new usages through the admin interface on the webpage at `<homepage_url>/admin`, by clicking `Usage revisions`, Import, and then choose the corresponding file from `<Xposition>/scripts/<corpus><version>_files`.
+	- Run `python manage.py shell` and then type `from scripts.generate_usage_file import main; main()`. This will create a json file for all usages in the corpus. 
+	- You can then import new usages through the admin interface on the webpage at `<homepage_url>/admin`, by clicking `Usage revisions`, Import, and then choose the corresponding file from `<Xposition>/xposition/scripts/<corpus><version>_files`.
 	- Note: Usages depend on construals and adpositions, so if there are any construals or adpositions missing from the database, you may get a 'Missing' error.
 - Import new sentences and ptoken_annotations:
-	- Run `python manage.py shell` and then type `exec(open(r'..\scripts\generate_corpus_files.py').read())`. This will create a json file for all CorpusSentences and PTokenAnnotations in the corpus. 
-	- You can then import them through the admin interface on the webpage at `<homepage_url>/admin`, by clicking `Corpus sentences` or `Adposition token annotations`, Import, and then choose the corresponding file from `<Xposition>/scripts/<corpus><version>_files`. Adposition token annotations will be divided into multiple files labelled `ptoken_annotations0.tsv`, `ptoken_annotations1.tsv`, etc.  
+	- Run `python manage.py shell` and then type `from scripts.generate_corpus_files import main; main()`. This will create a tsv file for all CorpusSentences and PTokenAnnotations in the corpus. 
+	- You can then import them through the admin interface on the webpage at `<homepage_url>/admin`, by clicking `Corpus sentences` or `Adposition token annotations`, Import, and then choose the corresponding file from `<Xposition>/xposition/scripts/<corpus><version>_files`. Adposition token annotations will be divided into multiple files labelled `ptoken_annotations0.tsv`, `ptoken_annotations1.tsv`, etc.  
 	- Note: PTokenAnnotations depend on usages and adpositions, so if there are any usages or adpositions missing from the database, you may get a 'Missing' error.
 - If you are importing a new version of an existing Corpus, please mark any older versions of the Corpus as deprecated, so that annotations from older versions will be hidden by default. Go to the corpus page and in the table titled "Metdata", click the edit button and check the box for "Is this a deprecated version of an existing Corpus".
```

Comment:
fix typo: Metdata

(I fixed this on GH and I think it is causing a merge conflict)

Reply:
There are some other changes in the README on both branches, but my branch is correct.